### ISSUE TITLE
Tilemap bug

### ISSUE THREAD

**Issue by rayandfz** [excaliburjs/Excalibur#1497]
```js
export class MirailleMap extends Scene {
    constructor(engine: Engine) {
        super(engine);
    }

    onInitialize(_engine: Engine) {
        this.add(Resources.MirailleMap.getTileMap())
    }
}
```js
const mirailleMap = require("./assets/images/Village.json")

let Resources = {
    MirailleMap: new TiledResource(mirailleMap),
}
```
Error : 
```js
Uncaught TypeError: Cannot read property 'tilesets' of null
```

I use : https://github.com/excaliburjs/excalibur-tiled

**Comment by DaVince** [comment]
I don't know if this will solve your problem, but you don't have to require the json file. Instead, try passing the path to the json file to TiledResource:

```typescript
const mirailleMap = "./assets/images/Village.json";
```

**Comment by Qarasina** [comment]
Hello, ty but i've another error : 
```
Access to XMLHttpRequest at 'file:///C:/Users/Rayan/Desktop/RubiaNova-Game/rubianova/dist/assets/images/Village.json?__=1587142517161' from origin 'null' has been blocked by CORS policy: Cross origin requests are only supported for protocol schemes: http, data, chrome, chrome-extension, https.
```

**Comment by DaVince** [comment]
That's an unrelated error. You can't run Excalibur directly from a local HTML file -- you need to host it on some sort of local server. This has to do with restrictions on local files that are blocked by browsers for security reasons.

**Comment by DaVince** [comment]
Hi @Qarasina, did you manage to get things working? 😃

**Comment by eonarheim** [comment]
@Qarasina can you post a small example of your .json file from tiled that fails? Or link to a repo with a reproduction of the issue?

**Comment by Carghaez** [comment]
Hello guys, I don't know if it is related to the same cross-origin issue, but I notice that on https://codesandbox.io/ if you try to use a version of excalibur above 0.23.0 (i.e. 0.24.x) you get
<< Error: Blocked a frame with origin "...." from accessing a cross-origin frame. >>

I admit that this is a bit frustrating because I often used Excalibur and codesanbox to show my students the potential of Typescript and coding as a playful and easy element to try

**Comment by kamranayub** [comment]
@Carghaez I think this is what I just filed #1510

**Comment by Qarasina** [comment]
Hello

I did have a web server up and running, though.

JSON File : https://hastebin.com/exazezuzab.json

**Comment by DaVince** [comment]
If I'm not mistaken this is now solved in 0.24.2 through issue #1510.

**Comment by eonarheim** [comment]
I believe this has been solved, please let me know if it hasn't and we will re-open
